### DISCUSSION TITLE
Table 组件 DisableExtendEditButtonCallback 在 CardView 模式下，不能正常进入回调

### DISCUSSION THREAD

**Discussion by Jiang2020426** [dotnetcore/BootstrapBlazor#3951] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Table组件当设置了DisableExtendEditButtonCallback回调，在Table模式下，可用正常进入回调，但是在CardView模式下，不能正常进入回调

### Expected Behavior

应该在CardView模式下也能进入回调，判断编辑按钮是否可用

### Steps To Reproduce

![1](https://github.com/user-attachments/assets/c2a9afaf-d233-47ee-ad0f-40fc24af7f92)
![2](https://github.com/user-attachments/assets/f4e31233-6435-45d9-ad87-e7824103ae93)
[EditButtonCallBackDemo.zip](https://github.com/user-attachments/files/16420976/EditButtonCallBackDemo.zip)
以上是项目截图和demo

### .NET Version

NET8

**Comment by bb-auto** [comment]
@Jiang2020426 Thank you for reporting. We will give triage later.

**Reply by ArgoZhang** [reply]
@Jiang2020426 `8.7.5-beta03` 已修复
